xonsh/xonfig.py

```diff
@@ -672,6 +754,7 @@ def _xonfig_create_parser():
     "styles": _styles,
     "colors": _colors,
     "tutorial": _tutorial,
+    "kernel": _kernel,
```

Comment:
Maybe this subcommand etc should be called `jupyter` to distinguish it from other kernels

Reply:
I was trying to distinguish this option from the other xonfig options and trying to keep it short.  How about option "jupyter_kernel"? I just noticed xonfig has completion, it's not too much to type.  (Though I hate introducing the mental confusion between snake case and kebab case...)

Reply:
I would be ok with `jupyter-kernel`. The important bit is that jupyter appears in the name

Reply:
ok